examples/lock-app/linux/src/LockManager.cpp

```diff
@@ -89,15 +89,25 @@ bool LockManager::InitEndpoint(chip::EndpointId endpointId)
         numberOfYearDaySchedulesPerUser = 10;
     }
 
+    uint8_t numberOfHolidaySchedules = 0;
+    if (!DoorLockServer::Instance().GetNumberOfHolidaySchedulesSupported(endpointId, numberOfHolidaySchedules))
+    {
+        ChipLogError(
+            Zcl,
+            "Unable to get number of supported holiday schedules when initializing lock endpoint, defaulting to 10 [endpointId=%d]",
+            endpointId);
+        numberOfYearDaySchedulesPerUser = 10;
```

Comment:
Why are we resetting `numberOfYearDaySchedulesPerUser` if holiday schedules are not supported?

Reply:
This was a typo, fixed here: https://github.com/project-chip/connectedhomeip/pull/18860